local-cli/eject/eject.js

```diff
@@ -84,6 +129,29 @@ function eject() {
       appName,
       templateOptions
     );
+
+    console.log('Setting up App Icons');
+    const hdpiPath = 'android/app/src/main/res/mipmap-hdpi/ic_launcher.png';
+    const mdpiPath = 'android/app/src/main/res/mipmap-mdpi/ic_launcher.png';
+    const xhdpiPath = 'android/app/src/main/res/mipmap-xhdpi/ic_launcher.png';
+    const xxhdpiPath = 'android/app/src/main/res/mipmap-xxhdpi/ic_launcher.png';
+
+    const pictureSizes = [48, 72, 96, 144];
```

Comment:
@ericvicenti I just updated it with your data structure proposal. Lemme know if this works for you!